account_financial_risk/models/res_partner.py

```diff
@@ -152,16 +150,16 @@ def _get_field_risk_model_domain(self, field_name):
     @api.depends(
         "invoice_ids",
         "invoice_ids.state",
-        "invoice_ids.amount_total_company_signed",
+        "invoice_ids.amount_total_signed",
         "child_ids.invoice_ids",
         "child_ids.invoice_ids.state",
-        "child_ids.invoice_ids.amount_total_company_signed",
+        "child_ids.invoice_ids.amount_total_signed",
     )
     def _compute_risk_invoice(self):
```

Comment:
You can remove this method adding "draft" key in _risk_account_groups()

Reply:
but the read_group for draft runs in account.move, not in account.move.line 🤔

Reply:
could you check last commit?

Reply:
As far I can see, doing the draft with the move lines instead of the account move means that the tax won't be included in the draft amount.

Reply:
Sorry, instead of 
('exclude_from_invoice_tab', '=', False),

Use in:
('account_internal_type', '=', 'receivable')